### PR TITLE
BC-434: Megamenu Frontend Integration

### PR DESCRIPTION
**Pull request by BenHarrington**
# Description

This PR includes the following proposed change(s):

- [BC-434](https://bcdevex.atlassian.net/jira/software/projects/BC/boards/6?selectedIssue=BC-434)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Does the change impact or break the Docker build?

- [ ] Yes
- [x] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [ ] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by siva-ka** [comment]
DEPRECATION WARNING: Using / for division is deprecated and will be removed in Dart Sass 2.0.0.

Recommendation: math.div(100, 3)

Could you please fix this? It looks great otherwise, thanks Ben!

**Review by siva-ka — DISMISSED** [review]
looks good, thanks

File: src/staging/src/components/header.js [hunk 1]
```diff
@@ -1,11 +1,13 @@
 import React from "react"
-import { css } from "@emotion/react"
-import { useStaticQuery, Link, graphql } from "gatsby"
+import MegaMenu from "./megaMenu.js"
```
Comment (BcGovNeal) [inline src/staging/src/components/header.js:2]: `".megaMenu.js"` filename is case sensitive

**Review by BcGovNeal — CHANGES_REQUESTED** [review]
On a first pass, please fix the React warnings in the console.  Such as the missing prop types and missing "key" prop warnings

File: src/staging/src/components/header.js (hunk 1 above)
Comment (BenHarrington) [reply src/staging/src/components/header.js:2]: Case sensitivity, warnings, and React errors have been fixed. I have went ahead and fixed most of the console warnings in and outside of my body of work but there a couple left:

- \components\park\mapLocation.js
- \components\park\parkHeader.js
- \pages\park-search.js

@siva-ka is working in this area and I do not want to step on her toes so I will leave that to her, or whomever picks up the Jira ticket about cleaning up errors/warnings.

**Review by BcGovNeal — APPROVED** [review]
Menu looks good!

**Review by siva-ka — APPROVED** [review]

**Pull request merged**